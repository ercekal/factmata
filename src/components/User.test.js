import React from 'react';
import User from './User';
import renderer from 'react-test-renderer';
import data from '../fixtures/userData.json'

it('renders correctly', () => {
  const tree = renderer
    .create(<User item={data[0]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});