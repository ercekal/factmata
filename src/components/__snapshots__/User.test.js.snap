// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bwzfXH ivwhAr"
  onClick={[Function]}
>
  <div
    className="sc-htpNat eXoYXI"
  >
    <p
      className="sc-ifAKCX hbpRLv"
    >
      Leanne Graham
    </p>
  </div>
  <p
    className="sc-EHOje cwbZiP"
  >
    @
    Bret
  </p>
</div>
`;
